src/runtime/threading_backend.cc

```diff
@@ -26,16 +30,7 @@ class ThreadGroup::Impl {
     for (int i = exclude_worker0; i < num_workers_; ++i) {
       threads_.emplace_back([worker_callback, i] { worker_callback(i); });
     }
-    const char *val = getenv("TVM_BIND_THREADS");
```

Comment:
TVM_BIND_THREADS is an env variable asking whether we should set affinity(or let threads run as they are). We should take this into consideration.

When TVM_BIND_THREADS is defined and it is 0, we should not do SetAffinity at all, and configure have no effect other than return the corresponding value

Reply:
Not sure what should be done here, as currently this branch is moved to ThreadPool; the behavior should be close to what it was before (now Configure is not called so SetAffinity is not called either, and the default num_workers_ is used).

Reply:
Currently `CHECK_EQ` will break in the case of x86 with hyperthreading when MaxConcurrency reports #logicalcores/2 and std::hardware_concurrency gives #logicalcores.